src/pages/dashboardGroups/GroupsPage.tsx

```diff
@@ -130,7 +123,14 @@ function GroupsPage({ party, activeProducts, productsMap }: Props) {
         </Grid>
       )}
       <Grid item xs={12} sx={{ height: '100%' }}>
-        <Grid container direction="row" alignItems={'center'}>
+        <Grid
+          container
+          direction="row"
+          alignItems={'center'}
+          sx={{ backgroundColor: '#EEEEEE' }}
```

Comment:
Isn't this color in mui-italia?

Reply:
no, but #F2F2F2 is very similar. I'll use this.